plugins/gestures.koplugin/main.lua

```diff
@@ -120,6 +121,22 @@ Multiswipes allow you to perform complex gestures built up out of multiple swipe
 
 These advanced gestures consist of either straight swipes or diagonal swipes. To ensure accuracy, they can't be mixed.]])
 
+-- If the gesture contains the "toggle_touch_input" action, mark it always_active
+-- to make sure that InputContainer won't block it after the IgnoreTouchInput Event.
+function Gestures:isGestureAlwaysActive(ges, multiswipe_directions)
+    -- Handle multiswipes properly
+    -- NOTE: This is a bit clunky, as ges comes from the list of registered touch zones,
+    --       while multiswipe_directions comes from the actual input event.
+    --       Alas, all multiswipes are handled in a single multiswipe zone.
+    if self.multiswipes_enabled then
+        if ges == "multiswipe" and multiswipe_directions then
+            ges = "multiswipe_" .. self:safeMultiswipeName(multiswipe_directions)
```

Comment:
(Probably peanuts vs the rest of things happening, but just asking: this (safeMultiswipeName() is done on each gesture, so not too costly ? although one gesture drawn per second is hard to do :)

Reply:
It happens once per InputContainer, as long as it registers a `multiswipe` zone (`ges` is `multiswipe`), *and* the input event was actually a multiswipe (otherwise `multiswipe_directions` is `nil`).

(It's a single gsub call that replaces spaces with underscores, FWIW).